database/scripts/lib/buildfarm_tools.rb

```diff
@@ -92,6 +90,29 @@ def self.test_regression_reported_issues(error_name, status: nil)
     is_known_issue.map { |issue| { 'github_issue' => issue['github_issue'], 'status' => issue['status'] } }.uniq
   end
 
+  def self.jobs_last_success
+    run_command('./sql_run.sh jobs_last_success.sql')
+  end
+
+  def self.jobs_never_passed
+    run_command('./sql_run.sh jobs_never_passed.sql')
+  end
+
+  def self.jobs_failing(days_exclude: 0)
+    # Keys: job_name, last_success
+    out = []
+    jobs_never_passed.each do |e|
+      out << {"job_name" => e["job_name"], "last_success" => "Never"}
+    end
+
+    jobs_last_success.each do |e|
+      last_success = DateTime.parse(e['last_success_time']) 
+      next if last_success > (Date.today - days_exclude)
```

Comment:
IIUC, the purpose of days_exclude is skipping any success in the last days_exclude days, right? Im thinking if the variable can be named something along the lines of "older_than_days"

Reply:
The function jobs_failing should reflect that you are filling the section of the report, a quick glance would make you think that the function returns all jobs that are failing